platform/core/src/services/DisplaySetService/DisplaySetService.ts

```diff
@@ -355,6 +358,65 @@ export default class DisplaySetService extends PubSubService {
         allDisplaySets.push(...displaySets);
       }
     }
+    if (allDisplaySets.length === 0) {
+      const imageSet = new ImageSet(instances);
```

Comment:
I don't like the fact that we are calling `ImageSet` inside displaySetService. Ideally we should not have this happen. My suggestion is to create another sopClassHandler for unsupported displaySets and if no sopHandler repsonded with DS then call that.

Reply:
I really don't need ImageSet. Calling just because of the id. If I create the id and return an object, it will be ok?

Reply:
I still think we need a new sopClassHandler that exists at the default extension and set on the displaySetService if any one wants to overwrite that then they can.